karma.conf.js

```diff
@@ -5,7 +5,7 @@ const loaders = require('./webpack/loaders');
 module.exports = function (config) {
   config.set({
     frameworks: [
-      'mocha',
+      'jasmine',
       'chai',
```

Comment:
Do we sti need chai and sinon since jasmine includes its own expectations and spies?